core/operations/management/eventing_get_status.cxx

```diff
@@ -48,7 +49,33 @@ eventing_get_status_request::make_response(error_context::http&& ctx, const enco
             response.error.emplace(problem);
             return response;
         }
-        response.status = payload.as<couchbase::core::management::eventing::status>();
+        response.status = payload.as<core::management::eventing::status>();
+        std::vector<core::management::eventing::function_state> filtered_functions{};
+        for (const auto& function : response.status.functions) {
+            bool include{};
+            if (bucket_name.has_value() && scope_name.has_value()) {
+                include = function.internal.bucket_name.has_value() && function.internal.scope_name.has_value() &&
+                          function.internal.bucket_name.value() == bucket_name.value() &&
+                          function.internal.scope_name.value() == scope_name.value();
+            } else {
+                include = (!function.internal.bucket_name.has_value() && !function.internal.scope_name.has_value()) ||
+                          (function.internal.bucket_name.has_value() && function.internal.scope_name.has_value() &&
+                           function.internal.bucket_name.value() == "*" && function.internal.scope_name.value() == "*");
+            }
+            if (include) {
+                CB_LOG_CRITICAL("Including function {} at {} / {}",
```

Comment:
Lets use CB_LOG_TRACE, or don't use logging here at all. After all we are logging the thing which is accessible in the response. If necessary, the caller can log it for us

Reply:
Oops, yes, this was for debugging only, it wasn't supposed to stay

Reply:
Removed 👍